app/helpers/application_helper.rb

```diff
@@ -51,4 +51,12 @@ def conditional_text(variable, prefix='', suffix='')
       prefix + variable.to_s + suffix
     end
   end
+
+  def prison_name_for_id(nomis_id)
+    @nomis_id_to_prison ||= \
+    Rails.configuration.prison_data.inject({}) do |h, (prison_name, prison_data)|
```

Comment:
Can this line be joined up with the one above more clearly? I realise the `\` is intended to do that, but the combination of indentation and line-length make this harder to scan and see the relationship. Maybe just make it one line, like this:

```ruby
@nomis_id_to_prison ||= Rails.configuration.prison_data.inject({}) do |h, (prison_name, prison_data)|
```

Alternatively, put `Rails.configuration.prison_data` in a variable?